source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -3759,7 +3759,11 @@ KX_PYMETHODDEF_DOC(KX_GameObject, rayCastTo,
 	const char *propName = "";
 	SCA_LogicManager *logicmgr = GetScene()->GetLogicManager();
 
-	if (!PyArg_ParseTuple(args,"O|fs:rayCastTo", &pyarg, &dist, &propName)) {
+    static const char *kwlist[] = {"other", "dist", "prop", nullptr};
+	if (!PyArg_ParseTupleAndKeywords(
+	        args, kwds, "O|fs:rayCastTo",  const_cast<char**>(kwlist),
+	        &pyarg, &dist, &propName
+	    )) {
```

Comment:
The code style we are using try to put all in one line if possible, if not you have to break and the `{` is not anymore in the same line than the condition but in a new line. Here i think you can put everything in the same line.